tools/enumerate-devices/rs-enumerate-devices.cpp

```diff
@@ -355,14 +355,16 @@ int main(int argc, char** argv) try
     }
 
     // Obtain a list of devices currently present on the system
-    json settings;
+    json settings = json::object();
```

Comment:
Should we update all occurrences of `json j;` to `json j = json::object()` or is it relevasnt only to the settings usage?

Reply:
Relevant only to the settings